conf/application.conf

```diff
@@ -64,9 +61,9 @@ auditing {
   enabled=true
 }
 
-timeoutDialog {
-    timeout-seconds = 900
-    timeout-countdown-seconds = 120
+session {
+    timeoutSeconds = 900
+    hmrc-timeout-dialog.defaultCountdownInSeconds = 120
```

Comment:
AppConfig won't find this key as it's under 'session', probably just worth making a custom one for this.